stable/object-bucket-claim/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for ObjectBucketClaim to be consumed by other applicat
 home: https://github.com/mesosphere/charts
 keywords:
   - kommander
-version: 0.1.0
+version: 0.1.1
 appVersion: "0.1.0"
```

Comment:
what should this appVersion be used for? if not tied to a version of something in particular, might as well have it match the chart version

Reply:
done